feg/protos/s8_proxy.proto

```diff
@@ -64,8 +64,6 @@ message CreateSessionRequestPgw {
     bytes indication_flag = 16;
     string charging_characteristics = 17;
     TimeZone time_zone = 18;
-
-    uint32 feg_relay_generated_c_agw_teid = 99;
```

Comment:
safe to remove. We just used temporarily to make sure teid was generated properly. Now we use CAgwTeid to inject the value